### ISSUE TITLE
Acr.UserDialogs crash on Android 5.0 and up (Xamarin.Forms)

### ISSUE THREAD

**Issue by Nyakoo** [aritchie/userdialogs#73]
Hi!

When using Acr.UserDialogs with Xamarin.Forms on a 5.0 device the app crashs on start. Replacing the package with Acr.UserDialogs.Android.AppCompat fixes the problem but that devices with versions under 5.0 are not compatible anymore.

Is there a fix for this or is this a problem with Xamarin.Forms (and thus the hint to not use AppCompat with Forms?)

**Comment by aritchie** [comment]
I'll need the stack trace please.  Appcompat will not work on Xamarin forms without additional libraries external to this one

**Comment by Nyakoo** [comment]
I tried to reproduce the crash but after installing the Acr.UserDialogs package again its just runs fine on 5.0+ devices and below.
Short order of what I did:

- Starting situation: Acr.UserDialogs installed. Runs fine on <5.0 but crashes on 5.0+ devices
- Uninstalled Acr.UserDialogs and all dependencies
- Installed Acr.UserDialogs.Android.AppCompat. Runs on 5.0+ but when uploading to devices with <5.0 it fails because of not compatible
- Uninstalled Acr.UserDialogs.Android.AppCompat and all dependencies
- Installed Acr.UserDialogs. Runs fine on all devices

Maybe something was wrong with the package at the beginning but I will keep an eye on this.
Just to confirm, normally Acr.UserDialogs should run without problems on 5.0+ devices, right?
